impl/src/main/java/org/apache/myfaces/cdi/view/ViewScopeBeanHolder.java

```diff
@@ -236,4 +236,8 @@ public String generateUniqueViewScopeId()
         return key;
     }
 
+    public void removeStorage(String viewScopeId){
```

Comment:
Maybe rename this to `removeStorageContextFromMap` instead?  

Let me know if you have a preference or any other suggestions.

Reply:
I think `removeStorage` is good.  Clean and concise